standards/04-community.md

```diff
@@ -0,0 +1,431 @@
+
+## Staying Informed
+
+### Release Announcements
+
+- [**Release Announcements**](https://github.com/overture-stack/docs/discussions/categories/announcements)
+    - To stay up-to-date with the latest Overture releases and updates, keep an eye on our Announcements category in our GitHub Discussions
+
+## Community Guidelines
+
+When participating in our community, please adhere to the following guidelines:
+
+1. Be respectful and inclusive of all community members.
+2. Provide as much relevant information as possible when asking for help.
+3. Use clear and descriptive titles for your discussions and issues.
+4. Search existing discussions before creating a new one to avoid duplicates.
+5. Follow the GitHub Community Guidelines and abide by our [code of conduct](/community/code-of-conduct).
+
+``````
+
+</details>
+
+## Team Page
+
+A showcase of the project's core team members, contributors, and acknowledgments. This page helps users connect with the people behind the project and recognizes those who have made significant contributions to its development.
+
+<details>
+<summary>**Click here for our templated support page**</summary>
+
+``````
+## Team Page
+
+<details>
+<summary>**Click here for our templated team page**</summary>
+
+# Team
+
+[Introduce your team, your expertise and your mission]
+
+## Authors
+
+<div class="row">   
+  <div class="col col--4 margin-bottom--lg margin-bottom--md">
+      <div class="card__header">
+        <div class="avatar avatar--vertical">
+                  <a href="https://github.com/user">
+          <img
+            class="avatar__photo avatar__photo--xl margin-bottom--md"
+            src="https://github.com/user.png"
+          />
+          </a>
+          <div class="avatar__intro">
+            <h3 class="avatar__name">Team Memeber</h3>
+          </div>
+        </div>
+    </div>
+  </div>
+
+
+## Honorary Mentions
+
+- **Name**: [Acknowledgement]
+- **Name**: [Acknowledgement]
+- **Name**: [Acknowledgement]
+- **Name**: [Acknowledgement]
+
+``````
+
+</details>
+
+## Funding Page
+
+A transparent overview of the financial support and organizations that make the project possible. This page acknowledges funding sources, grants, and additional support while providing information about project sustainability and institutional backing.
+
+<details>
+<summary>**Click here for our templated funding page**</summary>
+
+``````
+# Funding
+
+We gratefully acknowledge the financial support that has made this project possible. Our work has been funded by the following organizations and grants:
+
+### [Funding Source]
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
```

Comment:
woof I had forgotten this grant... writing the proposal for that was a doozy

Reply:
haha this is on the [deployment checklist](https://github.com/overture-stack/docs/issues/10#:~:text=Configure%20Netlify-,Update%20Funding%20Page,-Update%20submodules%20to), will have it completed by end of week